servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/AbstractQuarkusEvents.java

```diff
@@ -51,17 +51,18 @@
 import org.projectnessie.model.ContentKey;
 import org.projectnessie.model.IcebergTable;
 import org.projectnessie.model.MergeResponse;
-import org.projectnessie.model.Operation;
+import org.projectnessie.model.Operation.Put;
 import org.projectnessie.model.Reference;
+import org.projectnessie.model.UDF;
 import org.projectnessie.server.events.fixtures.MockEventSubscriber;
 
 @SuppressWarnings("resource")
 @ExtendWith(QuarkusNessieClientResolver.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public abstract class AbstractQuarkusEvents {
 
-  private final Operation.Put put =
-      Operation.Put.of(ContentKey.of("key1"), IcebergTable.of("somewhere", 1, 1, 3, 4));
+  private final Put putSource = Put.of(ContentKey.of("foo"), UDF.of("foo", "bar"));
+  private final Put putTarget = Put.of(ContentKey.of("key1"), IcebergTable.of("somewhere", 1, 1, 3, 4));
```

Comment:
The naming is a bit misleading imho since both operations are committed into `source`, but I'm fine leaving this as is.